README.md

```diff
@@ -33,20 +33,29 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the difference between an adaptive website and a fully responsive website?
+   A fully responsive design makes web pages render well on different devices and screen sizes.An adaptive website will work in all case scenarios i.e different displays but wont have the effect of a responsive design.
 
 2. Describe what it means to be mobile first vs desktop first.
 
+Mobile first means you start with the minimum width percentage and increase to the max width percentage. Desk top first means you start with a maximum width and reduce to the min width.
+
 3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units?
 
+It means that a coder uses 62.5% of the default settings for a webpage. That is 62.5% of HTML.
+
```

Comment:
exactly, the benefit of this is the default web browsers settings is 16px and 62.5% of that 16px is 10, bringing the base calculations to a base of 10, which is a lot easier to convert rems to the px base that is common, i.e. to achieve a font size of 16px when we have a base of 10 we are calculating with, the rem units will be 1.6, we just need to add a decimal at the 10s place to convert pxs to rems and benefit from the responsive nature of the rems units